ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/termsandconditions/v1/termsandconditions/_cq_template.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0"
+  jcr:primaryType="nt:unstructured"
+  jcr:title="Terms And Conditions"
+  fieldType="panel"
+  fd:tnc="{Boolean}true"
+  wrapData="{Boolean}true">
+  <text
+    name="consenttext"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="\0"
+    sling:resourceType="core/fd/components/form/text/v1/text"
+    fieldType="plain-text"
+    hideTitle="true"
+    value="Text related to the terms and conditions come here"/>
+  <approvalcheckbox
+    name="approvalcheckbox"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="I agree to the terms &amp; conditions"
+    sling:resourceType="core/fd/components/form/checkbox/v1/checkbox"
+    enabled="false"
+    checkedValue="true"
+    fieldType="checkbox">
+    <fd:events
+      jcr:primaryType="nt:unstructured"
+      custom_scrollDone="\{enabled: true()}"
```

Comment:
Why is this exposed via rule editor, shouldn't we hardcode in the view when view itself is dispatching this event ? If someone forgets this while creating proxy, the whole TnC would be broken ? I would preview to do this via subscribe in the view layer itself